content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Configurando um Pod Para Usar um Volume Projetado Para Armazenamento
+content_type: task
+weight: 70
+update_date: 2022-07-18
+origin_version: 1.24
+contributors: DonatoHorn
```

Comment:
Git tracks the date of changes and who made them. These fields don't look necessary.

Reply:
Hi, I'm new contributor on pt-br docs, so my difficults are identify pages (original version) than where used base-line for internacionalizations, in pt-br docs we have pages from v1.18, 1,19, 1.20, 1.21, 1.22, 1.23, 1.24, how can I guess what pages need update. I'ts nice to do this, make a diff in the english docs, so the pages that need update will be shown. But in pt-br we don't have versioning, so if I have a origin_version  in the docs I can use a grep to show what a version used in, and assign the contributor to ask if interested in update the docs they translated.

Reply:
That might be a nice enhancement to make. However, before you do this, we should get consensus between the localization teams to adopt that approach.

I'm afraid that changing this unilaterally is likely to be overall unhelpful.

Reply:
I don't  know who I need to talk about.